app/models/build.js

```diff
@@ -129,7 +129,7 @@ Build.reopen({
   },
 
   restart() {
-    return this.get('ajax').post("/builds/" + (this.get('id')) + "/restart");
+    return Ember.$.post(`/builds/${this.get('id')}/restart`);
```

Comment:
oh yeah, this ended up being a red herring when debugging why the test wasn’t working. It can be rolled back.